src/vdom/diff.js

```diff
@@ -244,7 +245,7 @@ function innerDiffNode(dom, vchildren, context, mountAll, isHydrating) {
 				if (f==null) {
 					dom.appendChild(child);
 				}
-				else if (child===f.nextSibling) {
+				else if (child===f.nextSibling || (l > 0 && child===originalChildren[i + l])) {
```

Comment:
1o1

Can we use another variable name?

Reply:
sure. Have you have suggestions?

Reply:
`childLength` or so.. :\

I mean, this is not even important :D